crates/rattler_conda_types/src/lib.rs

```diff
@@ -16,6 +16,7 @@ mod version_spec;
 
 pub mod package;
 pub mod prefix_record;
+pub mod virtual_package;
```

Comment:
I would just name this `generic_virtual_package` make it private and add `pub use generic_virtual_package::GenericVirtualPackage` in this file.

Reply:
Right, will do